tests/test_simulators.py

```diff
@@ -0,0 +1,59 @@
+#   Copyright 2022 Entropica Labs
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import unittest
+from openqaoa.workflows.optimizer import QAOA
+from openqaoa.problems.problem import QUBO
+from openqaoa.devices import create_device
+
+
+class TestSimulators(unittest.TestCase):
+    
+    def test_job_ids(self):
+
+        #define problem
+        problem = QUBO.random_instance(3)
+
+        #loop over all simulators
+        for n in ['qiskit.qasm_simulator', 'qiskit.shot_simulator', 'qiskit.statevector_simulator', 'vectorized', 'pyquil.statevector_simulator']:
```

Comment:
I think we should also have the tests for the QVM too

Reply:
done